main.js

```diff
@@ -10,3 +10,130 @@ document.querySelectorAll('.menu-link').forEach((n) => n.addEventListener('click
   hamburger.classList.remove('active');
   toolbarMenu.classList.remove('active');
 }));
+
+const popupSection = document.querySelector('.popupSection');
+const X = document.querySelector('.X');
+const page = document.querySelector('.page');
+
+document.querySelectorAll('.seeProject').forEach((n) => n.addEventListener('click', () => {
+  popupSection.style.display = 'block';
+  page.style.filter = 'blur(5px)';
+}));
+
+function disappear() {
+  popupSection.style.display = 'none';
+  page.style.filter = 'blur(0px)';
+}
+X.addEventListener('click', disappear);
+
+function disappearPage(event) {
+  if (event.target.className === 'popupSection') {
+    popupSection.style.display = 'none';
+    page.style.filter = 'blur(0px)';
+  }
+}
+popupSection.addEventListener('click', disappearPage);
+
+const projects = [
+  {
+    id: 0,
+    name: 'Multi-Post Stories <br> Gain+Glory',
+    description: 'Some description',
+    image: 'Assets/Img Plaholder.png',
+    technology: ['Ruby', 'CSS', 'JavaScript', 'HTML'],
+    source: '',
+    live: '',
+  },
+  {
+    id: 1,
+    name: 'Title 2',
+    description: 'Some description',
+    image: 'Assets/Img Plaholder.png',
+    technology: ['Ruby', 'CSS', 'JavaScript', 'HTML'],
+    source: '',
+    live: '',
+  },
+  {
+    id: 2,
```

Comment:
- [ ] Your popup appears well. But like the previous reviewer noted, kindly ensure it's impossible for a user to scroll through your application when the popup comes up, if not is looses the value of trying to show a particular project, and you're scrolling around. Please kindly make the popup fixed like in the original design and so it can serve its purpose.
![Screenshot (1157)_LI](https://github.com/GhausAliKhan/My-Portfolio/assets/65298048/d509090d-97bd-4152-8869-fb0e3d74981e)